src/firebase/models/players_in_room.js

```diff
@@ -0,0 +1,34 @@
+import { ref, onValue, onDisconnect, set, child } from "firebase/database";
+import rtdb from "../rtdb";
+
+const rooms = ref(rtdb, "rooms");
+const amOnline = ref(rtdb, ".info/connected");
```

Comment:
Establish amOnline as a variable which is a reference to the connection info in the realtime database